plugins/in_syslog/syslog_prot.c

```diff
@@ -104,7 +104,10 @@ int syslog_prot_process(struct syslog_conn *conn)
             flb_free(out_buf);
         }
         else {
-            flb_warn("[in_syslog] error parsing log message");
+            flb_warn("[in_syslog] error parsing log message "
+                    "(input_plugin_alias_name='%s' parser_name='%s')",
```

Comment:
- to gather the proper plugin name you can use the function [flb_input_name()](https://github.com/fluent/fluent-bit/blob/master/src/flb_input.c#L299), take a look at it prototype and do the proper replacement.

- I would recommend the message to be like "error parsing log message on "PLUGIN_NAME" with parser 'PARSER_NAME'.  (no parenthesis)